crates/debug/src/transform/expression.rs

```diff
@@ -604,6 +602,22 @@ where
                     // Don't re-enter the loop here (i.e. continue), because the
                     // DW_OP_deref still needs to be kept.
                 }
+                Operation::Shr { .. } | Operation::Shra { .. } => {
+                    // Insert value normalisation part.
```

Comment:
Can you expand on "normalisation" here? I recommend just include this PR description-like text here.

Reply:
Done: a806b96